Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertBlock.cs

```diff
@@ -1,190 +1,403 @@
+    // This prevents other conversions from being polluted by multi-document environments.
+    var converter = new ConverterRevit();
+    converter.SetContextDocument(famDoc); // Always remember to set the doc 🙂
+
+    // Using the family document, add all geometry of the instance at the root level (no nesting)
+    using DB.Transaction t = new(famDoc, $"Create geometry for block definition");
+    t.Start();
+    foreach (var o in geometry)
+      converter.ConvertToNativeObject(o);
+    t.Commit();
+  }
+
+  /// <summary>
+  /// Attempts to find the corresponding family symbol for a given block definition.
+  /// In general, since we're converting from a block to a family, we expect it to only have one symbol.
+  /// For this reason, current implementation will return the first symbol of the family.
+  /// </summary>
+  /// <param name="definition">The block definition to find a symbol for.</param>
+  /// <param name="family">The family that corresponds to th is block definition.</param>
+  /// <returns>The first family symbol of the family (as it is currently implemented).</returns>
+  private DB.FamilySymbol? FindBlockDefinitionFamilySymbol(BlockDefinition definition, DB.Family family)
+  {
+    // The loaded family contains all the possible symbols (variations)
+    // We must pick the right one and return.
+    // TODO: For now, we're just picking the first.
+    var element = Doc.GetElement(family.GetFamilySymbolIds().First());
+    if (element is not DB.FamilySymbol symbol)
+      return null;
+    if (!symbol.IsActive)
+      symbol.Activate();
+    return symbol;
+  }
+
+  /// <summary>
+  /// Creates a family symbol for the given block definition.
+  /// THIS IS NOT SUPPORTED YET
+  /// </summary>
+  /// <param name="definition">The block definition to create a symbol out of</param>
+  /// <param name="family">The family where the symbol will be created</param>
+  /// <returns>The new family symbol instance.</returns>
+  /// <exception cref="NotImplementedException">Throws always, as this is not implemented yet.</exception>
+  private DB.FamilySymbol CreateBlockDefinitionFamilySymbol(BlockDefinition definition, DB.Family family)
+  {
+    throw new NotImplementedException("Creating family symbols is not supported yet");
+  }
+
+  /// <summary>
+  ///   Creates a new family document as a duplicate of the specified template.
+  /// </summary>
+  /// <param name="name">The name of the template to use to create the family document.</param>
+  /// <returns>A new family document based based on the specified template.</returns>
+  /// <exception cref="System.IO.FileNotFoundException">When the file corresponding to the provided template name could not be found</exception>
+  /// <exception cref="Autodesk.Revit.Exceptions.InvalidOperationException">When the document could not be opened</exception>
+  private DB.Document CreateNewFamilyTemplateDoc(string name = "Block")
+  {
+    var templatePath = GetTemplatePath(name);
```

Comment:
I think we need to add these new rvts to the installer as well!

Reply:
Oh! So true.... will do so now

Reply:
Not necessary, all templates are grabbed with a wildcard
https://github.com/specklesystems/speckle-sharp-ci-tools/blob/fd65d9a7595cddafb63154e5af5dacfc5f2384b8/revit.iss#L82